sematic/cli/run.py

```diff
@@ -10,8 +10,11 @@
 # Sematic
```

Comment:
This file change is here for illustrative and reproducibility purposes. It will be removed before merging. Its contents will be included in a dedicated PR that exposes the Build System to users.